sktime/datasets/base.py

```diff
@@ -639,4 +639,4 @@ def load_uschange(y_name="Consumption"):
     if y_name != "Quarter":
         data = data.drop("Quarter", axis=1)
     X = data.drop(y_name, axis=1)
-    return y, X
+    return X, y
```

Comment:
Check out the `load_longley` function above in the same file, we need to make sure both are consistent in terms of input arguments and output.

Reply:
We now also support date/time indexes so we can replace the `pd.Int64Index` above with a `pd.PeriodIndex`

Reply:
I didn't see this, will change load_longley to be consistent with the rest. _load_dataset in base.py uses the X, y order as well.

Reply:
But that's classification, I think it makes more sense to have y, X no, because you always have the y in forecasting but not always X?

Reply:
Maybe. I don't think it's that big of an issue, but y, X is fine by me as well. However I think the important part is to choose one order and apply this through the entire package. We shouldn't have classification, regression and forecasting use different orders. Also internal class methods should hold this order.

Reply:
I don't agree, we follow sklearn for classification/regression and use X, y. For forecasting, because there's isn't always an X, it doesn't make sense to have X, y because that would change the first output if X isn't there, so I suggest to have y, X for forecasting.